### ISSUE TITLE
A11y_MySQLextensionforAzureDataStudio_NewDatabase_ScreenReader: ORCA Screen reader is announcing "Invalid Entry" when we navigate to 'Name' required field which is blank.

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio-mysql#171]
"[[Check out Accessibility Insights]((https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0))!]  - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### #GitHub Tags:
#A11yMAS; #A11yWCAG2.1; #A11ySev2; #A11yTCS; #DesktopApp; #GH_MySQLextensionforAzureDataStudio_Linux_Mar23; #MySQL extension for Azure Data Studio; #WCAG1.3.1; #Linux; #Element:EditField; #ScreenReader; #ORCA;

### Actual Result:​
ORCA Screen reader is announcing "Invalid Entry" when we navigate to 'Name' required field which is blank.

### Expected Result:​
ORCA Screen reader should not announce unnecessary "Invalid Entry" when we navigate to 'Name' required field which is blank.

### Environment Details:​
Application Name: MySQL extension for Azure Data Studio
Application Version: 1.42.0-insider
Operating System: Ubuntu 20.04.2 LTS
GNOME Version: 3.36.8
ORCA: 42.2-1ubuntu1
Distributor ID: Ubuntu
Description: Ubuntu 22.10
Release: 22.10
Codename: kinetic

### Repro Steps:

1. Launch Azure Data Studio application.
2. TAB to "Servers" and select any server from the list and hit ENTER on 'Connect' button.
3. TAB to connected server and right click on it and TAB to "Manage" and hit ENTER key.
4. TAB to "New Dashboard" and hit ENTER key.
5. "New Dashboard" screen gets open. Start the ORCA screen reader.
6. Observe that ORCA screen reader is announcing "Invalid Entry" when we navigate to 'Name' required field which is blank.

### User Impact:
Screen reader users will get confused about the announcement as they might have not entered any data but it is announcing as invalid entry.

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver."**

### Attachment:

[Github bug for invalid entry.webm](https://user-images.githubusercontent.com/90901519/228763873-1fb84cf6-4a42-41bc-9af6-6f01777a7ff5.webm)
